app/views/layouts/partials/_top_customers.html.erb

```diff
@@ -0,0 +1,6 @@
+<h3>Top 5 Customers</h3>
+<% @merchant.top_5_customers.each do |customer| %>
```

Comment:
In your Dashboard controller you defined @top_five_customers but in this  partial you don't use it, so either it can be implemented here or removed from the controller probably.

Reply:
Got rid of it in the controller. Thanks!!